src/OrleansRuntime/Placement/RandomPlacementDirector.cs

```diff
@@ -4,11 +4,11 @@
 
 namespace Orleans.Runtime.Placement
 {
-    internal class RandomPlacementDirector : PlacementDirector
+    internal class RandomPlacementDirector : IPlacementDirector<RandomPlacement>
     {
         private readonly SafeRandom random = new SafeRandom();
 
-        internal override async Task<PlacementResult> OnSelectActivation(
+        public virtual async Task<PlacementResult> OnSelectActivation(
```

Comment:
Because there are subclasses of RandomPlacementDirector which override these methods